Team54/src/main/java/ISA/Team54/users/controller/AuthenticationController.java

```diff
@@ -69,24 +72,105 @@
 		String jwt = tokenUtils.generateToken(user.getUsername());
 		int expiresIn = tokenUtils.getExpiredIn();
 
+
+		UserRole userRole = null;		
+		
+		if(authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_PATIENT"))) {
+			userRole = UserRole.PATIENT;
+		} else if (authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_SUPPLIER"))) {
+			userRole = UserRole.SUPPLIER;
+		} else if (authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_ADMIN"))) {
+			userRole = UserRole.ADMIN;
+		} else if (authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_SYSTEM_ADMIN"))) {
+			userRole = UserRole.SYSTEM_ADMIN;
+		} else if (authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_DERMATOLOGIST"))) {
+			userRole = UserRole.DERMATOLOGIST;
+		} else if (authentication.getAuthorities().stream()
+		          .anyMatch(r -> r.getAuthority().equals("ROLE_PHARMACIST"))) {
+			userRole = UserRole.PHARMACIST;
+		}
```

Comment:
Jesi siguran da ovo ne moze lakse? Tipa da samo odes u authorities tabelu i za id od korisnika vratis njegovu rolu

Reply:
pokusvalai smo milena i ja to dosta i nije moglo,pa sam ovako nasao na netu